lang/java/reef-checkpoint/src/main/java/org/apache/reef/io/checkpoint/CheckpointNamingService.java

```diff
@@ -24,10 +24,10 @@
 public interface CheckpointNamingService {
 
   /**
-   * Generate a new checkpoint Name.
+   * Generates a file name for a new checkpoint.
```

Comment:
The names don't have to refer to files on HDFS. Hence, the original comment seems better.